kibana.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "opendistroAlerting",
+  "version": "1.11.0.2",
+  "kibanaVersion": "kibana",
```

Comment:
Specify kibana version as "kibana" will auto resolve as 7.10?

Reply:
Hi Yaliang, thanks a lot for reviewing my PR. In my mind, it will resolve as the version defined in `package.json`. Let me find some references.

Reply:
It comes from "kbn-plugin-generator", in the template of the sample Kibana plugin, the value of "kibanaVersion" is set to "kibana". (https://github.com/elastic/kibana/blob/master/packages/kbn-plugin-generator/template/kibana.json.ejs).
As far as I see, it will get the version number from `package.json`.
Unfortunately, there is not much explanation for the manifest from Kibana. https://github.com/elastic/kibana/blob/master/docs/development/core/server/kibana-plugin-core-server.pluginmanifest.md

Reply:
Got it, it's ok if the build and yarn start test pass

Reply:
Thank you.👍 For the "kibanaVersion", probably it's not getting the version from "package.json" 🤔, because when I ran `yarn build`, the script prompts me to input the Kibana version manually.

Reply:
Interesting, you may check @yizheliu-amazon 's build changes [here](https://github.com/yizheliu-amazon/anomaly-detection-kibana-plugin/commit/2237078868dd4f8f1c57b496ef4496f784c857e2) - looks like you will want to change the `serverSourcePattern` in `.kibana-plugin-helpers.json`, and specify the plugin and kibana version in the Kibana manifest file. Not sure how the version verification and build naming has changed, with both the npm `package.json` and the Kibana manifest `kibana.json` specifying similar things now.

Reply:
Hi Tyler, thank you for your reminder. I didn't notice there is the name change in `.kibana-plugin-helpers.json`. 👍

Reply:
I will change the "kibanaVersion" to the exact version number to avoid inputting version number manually when building through `plugin-hlepers` in another PR for ODFE 1.12.

Reply:
Cool. Also, per the new naming convention, can you change `opendistroAlerting` to `opendistroAlertingKibana`?

Reply:
Thanks for pointing out. Sure, I just changed it in `kibana.json` in another branch, and will be in the following PR.